src/index.js

```diff
@@ -1,26 +1,80 @@
-import './styles/main.css'
-
+import './styles/main.css';
 
 const taskContainer = document.querySelector('.all-tasks');
 const createTask = document.getElementById('create');
 const formInput = document.querySelector('.input');
 const form = document.querySelector('form');
-let tasks = [];
+const tasks = [];
```

Comment:
According to the project requirements, by default, your app should contain a hardcoded array with some simple to-do tasks containing three keys. Then the tasks are used to populate a HTML list item. Please update your application to include this feature.